src/main/kotlin/io/sip3/captain/ce/pipeline/RtcpHandler.kt

```diff
@@ -18,37 +18,235 @@ package io.sip3.captain.ce.pipeline
 
 import io.sip3.captain.ce.Routes
 import io.sip3.captain.ce.USE_LOCAL_CODEC
-import io.sip3.captain.ce.domain.ByteArrayPayload
 import io.sip3.captain.ce.domain.Packet
-import io.sip3.commons.util.getBytes
+import io.sip3.captain.ce.domain.RtcpReportBlock
+import io.sip3.captain.ce.domain.RtcpSession
+import io.sip3.captain.ce.domain.SenderReport
+import io.sip3.commons.domain.SdpSession
+import io.sip3.commons.domain.payload.Encodable
+import io.sip3.commons.domain.payload.RtpReportPayload
 import io.vertx.core.Vertx
+import mu.KotlinLogging
+import kotlin.experimental.and
 
 /**
  * Handles RTCP packets
  */
 class RtcpHandler(vertx: Vertx, bulkOperationsEnabled: Boolean) : Handler(vertx, bulkOperationsEnabled) {
 
-    private val packets = mutableListOf<Packet>()
+    private val logger = KotlinLogging.logger {}
+
+    companion object {
+
+        const val MAX_VALID_JITTER = 10_000
+
+        const val R0 = 93.2F
+        const val I_DELAY_DEFAULT = 0.65
+        const val MOS_MIN = 1F
+        const val MOS_MAX = 4.5F
+    }
+
     private var bulkSize = 1
+    private var expirationDelay: Long = 4_000
+    private var aggregationTimeout: Long = 30_000
```

Comment:
Let's come to a convention. I never was in favour of underscores in numbers `4_000`. I would love to have just numbers without it.

Reply:
Sometimes counting zeros in numbers is really annoying. Previously in case of milliseconds i use pattern `seconds * 1000` to improve readability. But it was in Java.